RotaryWheelControl/RotaryWheel.xaml.cs

```diff
@@ -143,6 +145,52 @@ public RotaryWheel()
             };
         }
 
+        private void SpinTo(int itemIndex, int durationInSec = -1)
+        {
+            Random r = new Random(DateTime.Now.Millisecond);
+            var angleFromYAxis = 360 - Angle;
+            SelectedItem = _pieSlices
+                .SingleOrDefault(p => p.StartAngle <= angleFromYAxis && (p.StartAngle + p.Angle) > angleFromYAxis);
+
+            int count = _pieSlices.Count;
+            int currIndex = _pieSlices.IndexOf(SelectedItem);
+            int fullSpin = itemIndex / count;
+            itemIndex = itemIndex % count;
+            int steps = currIndex-itemIndex;
```

Comment:
Consider not modifying the `itemIndex` value as it would then no longer be "item index". Instead,

`int steps = currIndex - itemIndex % count;`